Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -67,6 +70,9 @@ open class MapView: UIView {
 
     private let cameraViewContainerView = UIView()
 
+    /// Holds ViewAnnotation views
+    private let viewAnnotationContainerView = SubviewInteractionOnlyView()
```

Comment:
I was considering to only add this view to the map when the `ViewAnnotationManager` is used. However it might not be a big performance improvement and would probably just complicate the setup, so I think it's easier to just leave it as it is.